### ISSUE TITLE
Models-1

### ISSUE THREAD

**Issue by kushi944100** [Learn-Write-Repeat/Tensorflow#13]
# You have to prepare a content file for this topic
## Interested ones can comment below this or you can text me on slack
- Linear Regression Case Study using TF

**Comment by SANJAY-NT** [comment]
I am Interested to work on this issue and I do have experience in ML and DL. I have implemented many projects using Tensorflow.
Is there a official template for issue assigning. 
I had one doubt will this add on project be counted as contribution.

**Comment by kushi944100** [comment]
@SANJAY-NT yeah for these you will have marks you can also check the labels for marks

**Comment by SANJAY-NT** [comment]
Sorry for delay I shall create the pull request in 2 more days.

**Comment by kushi944100** [comment]
yeah @SANJAY-NT
